### PR TITLE
Landing Page For Events We Attend

### PR DESCRIPTION
**Pull request by ceoger**
### Description
create this page so we can share about LBRY with people we meet in events

### Fixes #

### CODE REVIEW

File: view/template/page/events.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <a href="#about" class="btn-primary btn-large">Tell Me More!</a>
+      </div>
+    </div>
+  </div>
+  <div class="cover cover-light" id="about">
+    <div class="content content-light content-readable">
+      <h3>Enter your email address to receive LBRY updates, news, and information for India's LBRYians!</h3>
+      <p>As an open source project, our development and future relies on our communities around the world, and we're delighted to see our community growing in India. Please enter your email below and we'll give you instructions to redeem the LBC as mentioned by our team.</p>
+      <?php echo View::render('mail/_subscribeForm', [
+        'tag' => 'events',
```
Comment (tzarebczan) [inline view/template/page/events.php]: Can we change this tag to something else? @ceoger  / @jsigwart . Maybe something more specific.

File: view/template/page/CUniversity.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <a href="#about" class="btn-primary btn-large">Tell Me More!</a>
+      </div>
+    </div>
+  </div>
+  <div class="cover cover-light" id="about">
+    <div class="content content-light content-readable">
+      <h3>Enter your email address to receive LBRY updates, news, and information for India's LBRYians!</h3>
+      <p>As an open source project, our development and future relies on our communities around the world, and we're delighted to see our community growing in India. Please enter your email below and we'll give you instructions to redeem the LBC as mentioned by our team.</p>
+      <?php echo View::render('mail/_subscribeForm', [
+        'tag' => 'vit',
```
Comment (tzarebczan) [inline view/template/page/CUniversity.php]: new CU tag?

File: view/template/page/events.php (hunk 1 above)
Comment (ceoger) [reply view/template/page/events.php]: not sure, this is not a specific event, there is an exchange launch happening in new delhi for which we plan to pass by, we may get to talk to the attendees about LBRY, so maybe we can use an existing landing page else please suggest a any?

File: view/template/page/CUniversity.php (hunk 2 above)
Comment (tzarebczan) [reply view/template/page/CUniversity.php]: What do you mean by ignore? Do you still need this landing page?

File: view/template/page/events.php (hunk 1 above)
Comment (tzarebczan) [reply view/template/page/events.php]: What's the purpose of tagging these people? Will you reach out to them? Maybe make a generic one like `india-other` for these one offs? Just didn't want a tag in our system called "events" - too generic.

File: view/template/page/CUniversity.php (hunk 2 above)
Comment (ceoger) [reply view/template/page/CUniversity.php]: yes please, i still need it, the pr has previously been merged

**Comment by tzarebczan** [comment]
I think we should also change the url to something more appropriate as well. Is the CU page no supposed to be merged? Might be better to create a new PR :)

**Pull request closed without merging**